sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.aest.controller.ts

```diff
@@ -132,12 +157,15 @@ export class AssessmentAESTController extends BaseController {
    * Triggers manual reassessment for an application.
    * Application cannot be archived or in any of the statuses 'Cancelled', 'Overwritten' or 'Draft' and original assessment must be in completed status.
    * @param payload request payload.
-   * @param applicationId application id.
+   * @param applicationId parent application id.
    * @returns id of the assessment created.
    */
   @Roles(Role.AESTManualTriggerReassessment)
   @Post("application/:applicationId/manual-reassessment")
-  @ApiNotFoundResponse({ description: "Application id not found." })
+  @ApiNotFoundResponse({
```

Comment:
Same set of comments

TODO:
Not found error message correction.
Call applicationId as parentApplicationId